ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/definition/package.scala

```diff
@@ -0,0 +1,79 @@
+package pl.touk.nussknacker.restmodel
+
+import io.circe.Decoder
+import io.circe.generic.JsonCodec
+import io.circe.generic.semiauto.deriveDecoder
+import pl.touk.nussknacker.engine.api.definition.{MandatoryParameterValidator, ParameterEditor, ParameterValidator}
+import pl.touk.nussknacker.engine.api.process.SingleNodeConfig
+import pl.touk.nussknacker.engine.api.typed.typing.TypingResult
+import pl.touk.nussknacker.engine.definition.TypeInfos.MethodInfo
+import pl.touk.nussknacker.engine.graph.evaluatedparam
+import pl.touk.nussknacker.engine.graph.node.NodeData
+import pl.touk.nussknacker.restmodel.displayedgraph.displayablenode.EdgeType
+
+package object definition {
+
+  @JsonCodec(encodeOnly = true) case class UIProcessObjects(nodesToAdd: List[NodeGroup],
+                                                            processDefinition: UIProcessDefinition,
+                                                            nodesConfig: Map[String, SingleNodeConfig],
+                                                            additionalPropertiesConfig: Map[String, UiAdditionalPropertyConfig],
+                                                            edgesForNodes: List[NodeEdges],
+                                                            defaultAsyncInterpretation: Boolean)
+
+  @JsonCodec(encodeOnly = true) case class UIProcessDefinition(services: Map[String, UIObjectDefinition],
+                                 sourceFactories: Map[String, UIObjectDefinition],
```

Comment:
Can you fix tabs so all variables will be on the same level? (I know that it was before)

Reply:
ok, done